drivers/SmartThings/zwave-switch/src/qubino-switches/qubino-relays/qubino-flush-2-relay/init.lua

```diff
@@ -11,43 +11,180 @@
 -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
+local st_device = require "st.device"
+local capabilities = require "st.capabilities"
+local capabilities_defaults = require "st.capabilities.defaults"
+--- @type st.zwave.CommandClass
+local cc = require "st.zwave.CommandClass"
+--- @type st.zwave.CommandClass.Basic
+local Basic = (require "st.zwave.CommandClass.Basic")({ version = 1 })
+--- @type st.zwave.CommandClass.SwitchBinary
+local SwitchBinary = (require "st.zwave.CommandClass.SwitchBinary")({ version = 1 })
+--- @type st.zwave.CommandClass.Meter
+local Meter = (require "st.zwave.CommandClass.Meter")({ version = 4 })
+--- @type st.zwave.CommandClass.SensorMultilevel
+local SensorMultilevel = (require "st.zwave.CommandClass.SensorMultilevel")({ version = 7 })
+--- @type st.zwave.constants
+local constants = require "st.zwave.constants"
 
-local QUBINO_FLUSH_2_RELAY_FINGERPRINT = {mfr = 0x0159, prod = 0x0002, model = 0x0051}
+local QUBINO_FLUSH_2_RELAY_FINGERPRINT = { mfr = 0x0159, prod = 0x0002, model = 0x0051 }
 
 local function can_handle_qubino_flush_2_relay(opts, driver, device, ...)
   return device:id_match(QUBINO_FLUSH_2_RELAY_FINGERPRINT.mfr, QUBINO_FLUSH_2_RELAY_FINGERPRINT.prod, QUBINO_FLUSH_2_RELAY_FINGERPRINT.model)
 end
 
 local function component_to_endpoint(device, component_id)
-    if component_id == "main" then
-      return { 1 }
-    elseif component_id == "extraTemperatureSensor" then
-      return { 3 }
-    else
-      local ep_num = math.floor(component_id:match("switch(%d)"))
-      return { ep_num and tonumber(ep_num)}
-    end
+  if component_id == "main" then
+    return { 0, 1, 2 }
```

Comment:
wouldn't this multicast to all these endpoints? I imagine you probably just want to remove the multi-component functions

Reply:
The component to endpoint function is set and called only by the parent device and this is used to multicast commands to all these endpoints from parent device. (During porting I've noticed that ep 0 represents "status" of the whole device, you can send only Get zwave commands to this endpoint if you want to switch on/off the device you need to send set commands to ep 1 and 2 also; Ep 1, 2 are the children devices you can send both Gets and Sets to these endpoints)